docs/kb/semgrep-cloud-platform/inline-pr-comments.md

```diff
@@ -0,0 +1,27 @@
+---
+tags:
+  - Semgrep CI
+  - PR comments
+---
+
+import MoreHelp from "/src/components/MoreHelp"
+
+# Why did the comments on a PR or MR not appear inline?
+
+When Semgrep comments on PR or MR findings, the comments are usually posted on the line of code where the finding is identified (inline). However, there are two common reasons why comments may not appear inline.
+
+## Avoiding excessive inline comments
+
+For rules with several findings in the same PR or MR, inline comments for every individual finding occupy a significant amount of space without providing much additional information. Therefore, if the same finding occurs five or more times in a PR or MR, the related comment is posted as a summary (overall) comment.
```

Comment:
```suggestion
For rules with several findings in the same PR or MR, inline comments for every individual finding occupy a significant amount of space without providing much additional information. Therefore, if the same finding occurs more than five times in a PR or MR, the related comment is posted as a summary (overall) comment.
```

per https://github.com/returntocorp/semgrep-app/blob/4d1d2eb1de6165f5fb926a21f4a24e8e263ed469/server/semgrep_app/cloud_platform/scm_comments/controllers/base_comments.py#L1004

Reply:
Ah yes thank you! I must have lost the < vs <= in memory translation.